### PR TITLE
small doc changes

### PR DESCRIPTION
**Pull request by jalvz**

### CODE REVIEW

File: docs/guide/index.asciidoc [hunk 1]
```diff
@@ -8,23 +8,19 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
...
-you can automatically collect detailed performance information from inside your service with only minor changes to your code.
-APM agents will automatically instrument your code and measure the response time for incoming requests.
-It also automatically measures what your service was doing while it was preparing the response.
-For example,
-external database queries are measured to make it easy to detect slow queries.
-It also measures calls to caches,
-external HTTP requests etc.
-This makes it possible to pinpoint and fix performance problems and unhandled errors quickly and with minimal effort.
+It uses Elasticsearch as its data store and allows you to monitor software services and applications in real time.
+Regardless of whether your code is best described as a service or as an application, this guide will use indistinctly
```
Comment (graphaelli) [inline docs/guide/index.asciidoc]: I suggest -> `this guide will indiscriminately use the word 'service'`

File: docs/guide/index.asciidoc [hunk 2]
```diff
@@ -51,38 +47,30 @@ APM automatically instruments most popular web frameworks,
...
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the pre-built,
-open source Kibana dashboards that come with APM Server.
+To visualize the data after it's sent to Elasticsearch, you can use the pre-built open source Kibana dashboards
+that come with APM Server; or the dedicated APM UI bundled in X-Pack.
 
 We designed Elastic APM to run on anything from a regular laptop to thousands of machines,
-and it's easy to get started.
+and it's easy to get started with.
```
Comment (graphaelli) [inline docs/guide/index.asciidoc]: I wouldn't add the `with`

File: docs/guide/index.asciidoc [hunk 3]
```diff
@@ -163,14 +151,13 @@ output.elasticsearch:
 The HTTP API exposed by APM Server listens on `localhost` and port 8200 by default.
 If you change the listen address from `localhost` to something that is accessible from outside of the machine,
 we recommend setting up firewall rules to ensure that only your own systems can access the API.
-Use path of the form `"unix:/path/to.sock"` to listen on unix domain socket.
 Alternatively,
 you can use the {apm-server-ref}/security.html[secret token and TLS] to secure access to APM Server API.
-
+You can configure APM Server to listen on a UNIX domain socket.
```
Comment (graphaelli) [inline docs/guide/index.asciidoc]: `Unix`

**Comment by jalvz** [comment]
implemented suggestions here: 7d48864acd86d548acd9a2b04bbe84684979331f

**Pull request merged**